src/NServiceBus.NHibernate.AcceptanceTests-Oracle/ConfigureNHibernatePersistence.cs

```diff
@@ -13,11 +13,11 @@ public void Configure(BusConfiguration config)
     {
         NHibernateSettingRetriever.AppSettings = () => new NameValueCollection
         {
-            {"NServiceBus/Persistence/NHibernate/connection.driver_class", "NHibernate.Driver.OracleDataClientDriver"},
+            {"NServiceBus/Persistence/NHibernate/connection.driver_class", "NHibernate.Driver.OracleManagedDataClientDriver"},
```

Comment:
The reason is that our TeamCity agent's don't the have native client installed. All other branches use the managed driver.